go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/kubeedge/beehive v0.0.0
 	github.com/kubeedge/viaduct v0.0.0
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/mattn/go-sqlite3 v2.0.3+incompatible
+	github.com/mattn/go-sqlite3 v1.14.18
```

Comment:
we can update this to newest version 1.14.22

Reply:
Now beego/beego version has been upgraded to v2 and go-sqlite3 version has been upgraded to 1.14.22